tools/vcli/bw/repl/repl.go

```diff
@@ -467,14 +467,14 @@ func planBQL(ctx context.Context, bql string, s storage.Store, chanSize, bulkSiz
 	pln, err := planner.New(ctx, s, stm, chanSize, bulkSize, w)
 	if err != nil {
 		msg := fmt.Errorf("planer.New failed failed; %v", err)
```

Comment:
Little typo here: planner
Also failed is written twice.

Reply:
Done, thank you Thiago. =]